### PR TITLE
chore: update token expiration hydra doc

### PR DESCRIPTION
**Pull request by tomekpapiernik**
This PR updates the document that describes the token TTL config for OAuth2 clients in Hydra. 

- Things have been rewritten slightly
- A sample cURL has been added (single client config)
- Doc made SEO-friendlier - URL slug update, more descriptive title

## Checklist

- [ ] I have read the [contributing guidelines](../blob/master/CONTRIBUTING.md) and signed the CLA.
- [ ] I have referenced an issue containing the design document if my change introduces a new feature.
- [ ] I have read the [security policy](../security/policy).
- [ ] I confirm that this pull request does not address a security vulnerability. 
      If this pull request addresses a security vulnerability, 
      I confirm that I got green light (please contact [security@ory.sh](mailto:security@ory.sh)) from the maintainers to push the changes.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] I have added necessary documentation within the code base (if appropriate).

### CODE REVIEW

File: docs/hydra/guides/client-token-expiration.mdx [hunk 1]
```diff
@@ -0,0 +1,205 @@
+---
+id: client-token-expiration
+title: Configure token expiration time
+sidebar_label: Configure instance-wide token expiration time or adjust it for individual OAuth2 clients
+---
+
+The Hydra `config.yaml` file located in the main Hydra directory allows you to adjust:
```
Comment (aeneasr) [inline docs/hydra/guides/client-token-expiration.mdx:7]: While we're at it :)

```suggestion
The Ory Hydra `config.yaml` file located in the main Ory Hydra directory allows you to adjust:
```

File: docs/hydra/guides/client-token-expiration.mdx [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+- Expiration time of the user login and consent flow
+- Expiration time for an access token
+- Expiration time for an ID token
+- Expiration time for auth codes
+
+This configuration is instance-wide, which means that it applies to all registered clients.
+
+If you need to set different token lifespans for specific clients, you can do that by calling the
+[OAuth2ClientLifespans](../reference/api#operation/UpdateOAuth2ClientLifespans) endpoint of the Hydra admin API. The client
```
Comment (aeneasr) [inline docs/hydra/guides/client-token-expiration.mdx:17]:
```suggestion
[OAuth2ClientLifespans](../reference/api#operation/UpdateOAuth2ClientLifespans) endpoint of the Ory Hydra Admin API. The client
```

File: docs/hydra/guides/client-token-expiration.mdx [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+configuration takes precedence over the instance-wide configuration. Read [this section](#client-token-lifespan-configuration) to
+learn more.
+
+## Instance-wide configuration
+
+You can change the configuration on the instance-wide level by adjusting specific keys in the Hydra configuration.
```
Comment (aeneasr) [inline docs/hydra/guides/client-token-expiration.mdx:23]:
```suggestion
You can change the configuration on the instance-wide level by adjusting specific keys in the Ory Hydra configuration.
```

File: docs/hydra/guides/client-token-expiration.mdx [hunk 4]
```diff
@@ -0,0 +1,205 @@
...
+
+The configuration applied to a specific client using this endpoint precedes the instance-wide configuration set in the Hydra
+configuration file.
+
+:::
+
+For example, to update the configuration of a client registered in a Hydra instance that runs locally, send this request:
+
+```shell
+$ curl -X PUT localhost:4445/clients/db779000-01f3-4c12-bf70-ff85933d485e/lifespans --data '{
```
Comment (aeneasr) [inline docs/hydra/guides/client-token-expiration.mdx:157]: Cool idea to use a curl command to show this!

I think this is missing the `Content-Type: application/json` header :) And the URL probably needs `http://` prefix

**Review by aeneasr — APPROVED** [review]

File: docs/hydra/guides/client-token-expiration.mdx [hunk 5]
```diff
@@ -0,0 +1,205 @@
+---
+id: client-token-expiration
+title: Configure token expiration time
+sidebar_label: Configure instance-wide token expiration time or adjust it for individual OAuth2 clients
```
Comment (aeneasr) [inline docs/hydra/guides/client-token-expiration.mdx:4]: 👍

**Comment by tomekpapiernik** [comment]
@aeneasr Applied your suggestions, hope it's okay now. PTAL :)

File: docs/hydra/guides/client-token-expiration.mdx (hunk 4 above)
Comment (tomekpapiernik) [reply docs/hydra/guides/client-token-expiration.mdx]: done

**Review by aeneasr — APPROVED** [review]
epic

**Pull request merged**